src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/_SearchByReferencePartial.cshtml

```diff
@@ -2,13 +2,15 @@
 
 @using (Html.BeginForm("Index", "ManageEvidenceNotes", FormMethod.Get))
 {
-<div class="govuk-!-width-full govuk-panel-grey-background">
-    <div class="govuk-form-group govuk-!-width-one-half govuk-!-margin-left-2 govuk-!-padding-top-2">
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.FilterViewModel.SearchRef)">
-            @Html.Gds().LabelFor(m => m.FilterViewModel.SearchRef, new { @class = "govuk-label--s" }, false)
-            @Html.Gds().TextBoxFor(m => m.FilterViewModel.SearchRef)
-            <button class="govuk-button"><i class="fa fa-search" aria-hidden="true"></i><span class="sr-only govuk-visually-hidden">Search by reference number</span></button>
+    @Html.Hidden("activeOverviewDisplayOption", ViewData["activeOverviewDisplayOption"])
+
+    <div class="govuk-!-width-full govuk-panel-grey-background">
+        <div class="govuk-form-group govuk-!-width-one-half govuk-!-margin-left-2 govuk-!-padding-top-2">
+            <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.FilterViewModel.SearchRef)">
+                @Html.Gds().LabelFor(m => m.FilterViewModel.SearchRef, new { @class = "govuk-label--s" }, false)
+                @Html.Gds().TextBoxFor(m => m.FilterViewModel.SearchRef)
+                <button class="govuk-button"><i class="fa fa-search" aria-hidden="true" title="search by reference"></i><span class="sr-only">Search by reference number</span></button>
```

Comment:
just wondering if there is a Gds control for it?

Reply:
Yeah will look at moving this into a control when we add the search to the other PCS screen